CONTRIBUTING.md

```diff
@@ -61,14 +61,17 @@ and [Apollo's Docs Theme for Gatsby](https://github.com/apollographql/gatsby-the
 .
 
 To contribute to these docs, you can add or edit the markdown & MDX files in the `docs/source` directory.
+<!-- 
+TODO: port @everlastingbugstopper s work on rover https://github.com/apollographql/rover/pull/1118
+and update this file
 
 To build and run the documentation site locally, you'll have to install the relevant packages by doing the following
 from the root of the `router` repository:
 
 ```sh
 cd docs
 npm i
-npm start
+npm start -->
```

Comment:
🤔

Reply:
the three steps defined above:

```
cd docs
npm i
npm start``` are out of date; We will turn this //TODO comment into an issue, which @SimonSapin wants to tackle as a followup

Reply:
I’ve submitted https://github.com/apollographql/router/pull/1036. Maybe undo this chunk here to avoid a merge conflict?